src/components/Menu/LanguageNavigation/LanguageNavigation.tsx

```diff
@@ -88,7 +88,7 @@ const LanguageNavigation = ({
         />
       ) : null}
 
-      {items.length >= 2 ? (
+      {items.length >= 1 ? (
```

Comment:
I can't really approve this without knowing why this change is being made, won't this show the language button on every code snippet on the site?

Reply:
@MymmiJ only if there is `languages` is not empty but if the code is just JSON, curl or bash the navigation will not be displayed. So far when I browse the pages its okay but I need to confirm with @m-hulbert  first if all okay.

Reply:
Cool!